<?xml version="1.0" encoding="UTF-8"?><testrun name="MultiplicationValidationTestException" project="Digit_Calculation" tests="2" started="2" failures="0" errors="0" ignored="0">
  <testsuite name="test.com.sviatlana.digit.validation.MultiplicationValidationTestException" time="0.001">
    <testsuite name="[0]" time="0.001">
      <testcase name="numberValidationTest[0]" classname="test.com.sviatlana.digit.validation.MultiplicationValidationTestException" time="0.001"/>
    </testsuite>
    <testsuite name="[1]" time="0.0">
      <testcase name="numberValidationTest[1]" classname="test.com.sviatlana.digit.validation.MultiplicationValidationTestException" time="0.0"/>
    </testsuite>
  </testsuite>
</testrun>
